packages/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -896,7 +897,7 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
       this._colorFillBars = this.props.colorFillBars!;
     }
 
-    const yMinMaxValues = getMinMaxOfYAxis(this._points, ChartTypes.LineChart);
+    const yMinMaxValues = getMinMaxOfYAxis(this._points, ChartTypes.LineChart, YAxisType.NumericAxis);
```

Comment:
can this be made default. I don't think y axis can be anything other than numeric for any other chart.

Reply:
Done